indigo/indigo/src/main/scala/indigo/shared/time/FPS.scala

```diff
@@ -16,3 +16,5 @@ object FPS:
     def toDouble: Double   = fps.toDouble
     def toSeconds: Seconds = Seconds(1.0d / fps.toDouble)
     def toMillis: Millis   = toSeconds.toMillis
+
+  given CanEqual[FPS, FPS] = CanEqual.derived
```

Comment:
`derives CanEqual` should give you this instance for free 🤔

Reply:
On an opaque type?

Reply:
Ah, didn't notice the opaque type, nevermind then 😶‍🌫️